crates/router/src/connector/globalpay.rs

```diff
@@ -131,6 +131,23 @@ impl ConnectorValidation for Globalpay {
             ),
         }
     }
+    fn validate_mandate_payment(
+        &self,
+        pm_type: Option<types::storage::enums::PaymentMethodType>,
+        pm_data: types::domain::payments::PaymentMethodData,
+    ) -> CustomResult<(), errors::ConnectorError> {
+        let mandate_supported_pmd = std::collections::HashSet::from([
+            PaymentMethodDataType::Card,
+            PaymentMethodDataType::PaypalRedirect,
+            PaymentMethodDataType::GooglePay,
+            PaymentMethodDataType::Ideal,
+            PaymentMethodDataType::Sofort,
+            PaymentMethodDataType::Eps,
+            PaymentMethodDataType::Giropay,
+            PaymentMethodDataType::MandatePayment,
```

Comment:
`MandatePayment` should not be part of this validation.

Please remove this wherever applicable.